elasticai/creator/nn/quantized_grads/fixed_point/__init__.py

```diff
@@ -1,13 +1,42 @@
-from .quantize_to_fixed_point import (
-    quantize_to_fxp_hte as quantize_to_fxp_hte,
+from .quantize_to_fixed_point import quantize_to_fxp_hte, quantize_to_fxp_stochastic
+from .autograd import factory_autograd_func
+from .module_quantization import factory_fxp_forwbackw_quantization_module, factory_fxp_forw_quantization_module
+
+from .module_quantization import (
+    QuantizeForwHTE,
+    QuantizeForwHTEBackwHTE,
+    QuantizeForwHTEBackwStochastic,
+    QuantizeForwStochastic,
+    QuantizeForwStochasticBackwStochastic,
 )
-from .quantize_to_fixed_point import (
-    quantize_to_fxp_hte_ as quantize_to_fxp_hte_,
+from .param_quantization import (
+    QuantizeParamSTEToFixedPointHTE,
+    QuantizeParamSTEToFixedPointStochastic,
+    QuantizeParamToFixedPointHTE,
+    QuantizeParamToFixedPointStochastic,
 )
-from .quantize_to_fixed_point import (
-    quantize_to_fxp_stochastic as quantize_to_fxp_stochastic,
-)
-from .quantize_to_fixed_point import (
-    quantize_to_fxp_stochastic_ as quantize_to_fxp_stochastic_,
-)
-from .two_complement_fixed_point_config import FixedPointConfigV2 as FixedPointConfigV2
+from .two_complement_fixed_point_config import FixedPointConfigV2
+
+from .param_quantization import (QuantizeParamSTEToFixedPointHTE as QuantizeTensorToFixedPointHTE,
+    QuantizeParamSTEToFixedPointStochastic as QuantizeTensorToFixedPointStochastic,
+ ) # adding this Modules again under a different name to make clear that it can be used for Tensor quantization.
```

Comment:
I suggest adding that to the short doc string instead and just export it once for this purpose.
If you can see that the different exports will change for different reasons, that'd be a better reason to export them like this. Though in that case explicitly subclassing should probably be preferred.